worlds/tunic/er_rules.py

```diff
@@ -381,9 +381,10 @@ def get_paired_portal(portal_sd: str) -> Tuple[str, str]:
     regions["Overworld"].connect(
         connecting_region=regions["Overworld Tunnel Turret"],
         rule=lambda state: state.has(laurels, player))
-    regions["Overworld Tunnel Turret"].connect(
-        connecting_region=regions["Overworld"],
-        rule=lambda state: state.has_any({grapple, laurels}, player))
+
+    # regions["Overworld Tunnel Turret"].connect(
```

Comment:
Is this change intentional? If so, maybe a  comment as to why this is commented out might be good for the future.

Reply:
It's intentional, same reason as several others: you always have access to Overworld, so it's not necessary to connect to Overworld. It's only there in case we add start rando later.
I'll add the comment still though